src/core/Lockfile_target.ml

```diff
@@ -1,25 +1,76 @@
-(* A lockfile to be scanned by a supply chain rule.
-   Should contain a list of all dependencies used by
-   a project, at exact versions. Can be attached to a code target,
-   or a target by itself.
-*)
 type t = {
-  lockfile : Fpath.t;
-  lockfile_kind : Lockfile_kind.t;
-  lazy_lockfile_content : string lazy_t;
-  lazy_lockfile_ast_and_errors : Dependency.t list lazy_t;
-  manifest_target : manifest_target option;
+  source : Source.t;
+      (** The source of the data as is relevant to the user. This could be, e.g., a
```

Comment:
if Source.mli has good comments, and has a good name, then you would not need to repeat this comment everywhere.
But you do because Source is not a good name, and it's not clear what is its use.

Reply:
origin: Origin.t; seems clearer to me.

Reply:
Done in 3f206d0db